src/components/Settings/Settings.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+
+import './Settings.scss';
+
+interface Props {
+  itemWidth: number,
+  setItemWidth: (value: number) => void,
+  IMAGE_COUNT: number,
+  frameSize: number,
+  setFrameSize: (value: number) => void,
+  step: number,
+  setStep: (value: number) => void,
+  animationDuration: number,
+  setAnimationDuration: (value: number) => void,
+}
+
+export const Settings: React.FC<Props> = ({
+  itemWidth,
+  setItemWidth,
+  IMAGE_COUNT,
```

Comment:
```suggestion
  imageCount: number,
  frameSize: number,
  setFrameSize: (value: number) => void,
  step: number,
  setStep: (value: number) => void,
  animationDuration: number,
  setAnimationDuration: (value: number) => void,
}

export const Settings: React.FC<Props> = ({
  itemWidth,
  setItemWidth,
  imageCount,
```
Here it would be better to name it using camel case as here it's not a constant with only one possible value. It may be changed.